adventistchurch/alps
# Adding production Grunt tasks.

@designerbrent Take a look here and let me know what you think. 

To test this:

- Check out this branch
- Run `npm install` to install the `mkdir` module
- Run `grunt prod` from the command line.

This should create a directory in the `css` and `js` directories. This number is based on the [`version`](https://github.com/adventistchurch/alps/compare/grunt-prod?expand=1#diff-35b4a816e0441e6a375cd925af50752cR12) variable.

I've also removed the themer and media query test files. These can be turned back on when active development resumes.
